telegram/utils/helpers.py

```diff
@@ -113,3 +120,45 @@ def mention_markdown(user_id, name):
     """
     if isinstance(user_id, int):
         return '[{}](tg://user?id={})'.format(escape_markdown(name), user_id)
+
+
+def _extract_urls_from_text(text):
+    """
+    Returns a list of urls from a text string.
+    URLs without a leading `http://` or `www.` won't be found.
+    """
+    out = []
+    for word in text.split(' '):
```

Comment:
Is there a specific reason to split on explicit spaces instead of any white space?
IMHO, calling `text.split():` would make more sense. Do you agree?